app/templates/base.html

```diff
@@ -1,68 +1,79 @@
     <link href="{{ url_for('static', path='/style.css') }}" rel="stylesheet">
 
-    <title>Calendar</title>
+    <title>PyLendar {% block title %}{% endblock %}</title>
     {% endblock %}
 </head>
+
 <body>
-    <nav class="navbar navbar-expand-lg bg-transparent">
-        <div class="container-fluid">
-            <a class="navbar-brand" href="/">Calendar</a>
-            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarText"
-                aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
-                <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarText">
-                <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-                    <li class="nav-item">
-                        <a class="nav-link active" aria-current="page" href="{{ url_for('home') }}">Home</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link active" aria-current="page" href="{{ url_for('profile') }}">Profile</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="#">Sign In</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="#">Sign Up</a>
-                    </li>
-                    <li class="nav-item">
-                        <!-- new button for the agenda page. should be available after login
-                        place will change later according to the web design -->
-                        <a class="nav-link" href="{{ url_for('agenda') }}">Agenda</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="{{ url_for('view_invitations') }}">Invitations</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="/search">Search</a> 
-                    </li>
-                </ul>
-            </div>
+    <div class="bg-gradient2 h-1"></div>
+    <div class="relative overflow-hidden">
+        <div class="relative pt-6 px-4 sm:px-6 lg:px-8">
+            <nav class="navbar navbar-expand-md navbar-light">
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggler"
+                    aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="navbar-brand">
+                    <a href="{{ url_for('home') }}" aria-label="Home">
+                        <img class="h-8 w-auto sm:h-10"
+                            src="{{ url_for('static', path='/images/icons/calendar-outline.svg') }}" alt="Logo">
```

Comment:
please use the [ionicons ](https://ionicons.com/)element and not the svg file. 
```
<ion-icon name="calendar-outline"></ion-icon>
```

Reply:
Thanks, sure it will be done

Reply:
@aviadamar was ionicons checked for performance? 
and where are we using popover?
![image](https://user-images.githubusercontent.com/32178925/107156363-88684100-6986-11eb-8efe-03403206eab8.png)

Reply:
![image](https://user-images.githubusercontent.com/32178925/107156683-75567080-6988-11eb-836d-a2e7d32fc605.png)

Reply:
the script is in the head as written ? 
about the popover, I have no idea, if you find out let me know.

Reply:
I modified the scripts to be in the head as stated and it solved the warning in console.
Nevertheless, it stores some items in cache which reduce performance.
I changed the icon in the logo (navbar-brand).
But I didn't manage to find a way to do it in the page's tab icon.
Do you know how to do it in the page's tab icon?

Reply:
@aviadamar - this change done
Please review